bookkeeper-http/vertx-http-server/src/main/java/org/apache/bookkeeper/http/vertx/VertxHttpServer.java

```diff
@@ -70,6 +70,9 @@ public boolean startServer(int port) {
             @Override
             public void bindHandler(String endpoint, VertxAbstractHandler handler) {
                 router.get(endpoint).handler(handler);
+                router.put(endpoint).handler(handler);
```

Comment:
interesting... I am wondering why it was not captured by our tests. /cc @jiazhai can you check do we have test coverage of `gc` endpoint?

Reply:
I think test only covers get and not the put. I will add test for triggering gc.